### PR TITLE
PXC-3285 Added pipeline jobs for PXC-5.6 MTR run.

### PR DESCRIPTION
**Pull request by rameshvs02**

### CODE REVIEW

File: pxc/jenkins/pxc56-pipeline.groovy [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+                    rm -f ${WORKSPACE}/VERSION-${BUILD_NUMBER}
+                '''
+            }
+        }
+        stage('Check out and Build Galera/PXB') {
+            parallel {
+                stage('Build Galera library') {
+                    agent { label 'docker' }
+                    steps {
+                        git branch: 'PXC-3285-add-pipeline-jobs', url: 'https://github.com/Percona-Lab/jenkins-pipelines'
```
Comment (hors) [inline pxc/jenkins/pxc56-pipeline.groovy:107]: set 'master' branch  for https://github.com/Percona-Lab/jenkins-pipeline

File: pxc/jenkins/pxc56-pipeline.groovy [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+                                    exit 1
+                                fi
+                            '''
+                        }
+                    }
+                }
+                stage('Build PXB23') {
+                    agent { label 'docker' }
+                    steps {
+                        git branch: 'PXC-3285-add-pipeline-jobs', url: 'https://github.com/Percona-Lab/jenkins-pipelines'
```
Comment (hors) [inline pxc/jenkins/pxc56-pipeline.groovy:145]: please use 'master' branch  for https://github.com/Percona-Lab/jenkins-pipeline

File: pxc/jenkins/pxc56-pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    project-type: pipeline
+    description: |
+        Do not edit this job through the web!
+    concurrent: true
+    pipeline-scm:
+      scm:
+        - git:
+            url: https://github.com/Percona-Lab/jenkins-pipelines.git
+            branches:
+            - 'PXC-3285-add-pipeline-jobs'
```
Comment (hors) [inline pxc/jenkins/pxc56-pipeline.yml:12]: please fix the branch

File: pxc/local/build-binary-pxb23 [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+#BOOST_VERSION=$(grep 'SET(BOOST_PACKAGE_NAME' ${SOURCEDIR}/cmake/boost.cmake | sed -re 's/.*([0-9]+_[0-9]+_[0-9]+).*/\1/')
+#wget_loop "boost_${BOOST_VERSION}.tar.gz" "http://downloads.sourceforge.net/boost/boost/${BOOST_VERSION//_/.}/boost_${BOOST_VERSION}.tar.gz"
+#
+#if [ -f "${SOURCEDIR}/cmake/googletest.cmake" ]; then
+#    GMOCK_VERSION=$(grep 'SET(GMOCK_PACKAGE_NAME' ${SOURCEDIR}/cmake/googletest.cmake | sed -re 's/.*([0-9]+[.][0-9]+[.][0-9]+).*/\1/')
+#else
+#    GMOCK_VERSION=$(grep 'SET(GMOCK_PACKAGE_NAME' ${SOURCEDIR}/unittest/gunit/CMakeLists.txt | sed -re 's/.*([0-9]+[.][0-9]+[.][0-9]+).*/\1/')
+#fi
+#
+#wget_loop "googletest-release-${GMOCK_VERSION}.zip" "https://github.com/google/googletest/archive/release-${GMOCK_VERSION}.zip"
```
Comment (hors) [inline pxc/local/build-binary-pxb23:58]: remove it  from file

File: pxc/local/build-binary-pxc56 [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+# ------------------------------------------------------------------------------
+mkdir -p ${1:-./build}
+WORKDIR=$(cd ${1:-./build}; pwd -P)
+INSTALL_DIR=${WORKDIR}/DESTDIR
+DOWNLOAD_DIR=${WORKDIR}/source_downloads
+mkdir -p ${INSTALL_DIR} ${DOWNLOAD_DIR}
+SOURCEDIR=$(cd ${2:-$(dirname $0)/../sources/pxc}; pwd -P)
+
+# extract pxb-2.3 and galera binaries
+#PXB23_WORKDIR=$(cd ${2:-./build/pxb23}; pwd -P)
```
Comment (hors) [inline pxc/local/build-binary-pxc56:33]: remove this line

File: pxc/local/build-binary-pxc56 [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+GALERA3_WORKDIR=$(cd ${2:-./build/galera3}; pwd -P)
+
+#PXB23_TAR=$(ls $PXB23_WORKDIR/{percona-xtrabackup-2.*.tar.gz,pxb23.tar.gz}  | awk -F'/' '{print $NF}')
```
Comment (hors) [inline pxc/local/build-binary-pxc56:36]: the same

File: pxc/jenkins/pxc56-pipeline.groovy (hunk 1 above)
Comment (rameshvs02) [reply pxc/jenkins/pxc56-pipeline.groovy]: done

File: pxc/jenkins/pxc56-pipeline.groovy (hunk 2 above)
Comment (rameshvs02) [reply pxc/jenkins/pxc56-pipeline.groovy]: done

File: pxc/jenkins/pxc56-pipeline.yml (hunk 3 above)
Comment (rameshvs02) [reply pxc/jenkins/pxc56-pipeline.yml]: done

File: pxc/local/build-binary-pxb23 (hunk 4 above)
Comment (rameshvs02) [reply pxc/local/build-binary-pxb23]: done

File: pxc/local/build-binary-pxc56 (hunk 6 above)
Comment (rameshvs02) [reply pxc/local/build-binary-pxc56]: done

**Pull request merged**